music/demos/visualizer.html

```diff
@@ -65,6 +65,13 @@ <h2>mm.PianoRollCanvasVisualizer</h2>
       <canvas id="canvas"></canvas>
     </div>
   </section>
+
+  <h2>mm.StaffSVGVisualizer</h2>
+  <section>
+    <div class="visualizer-container" id="container3">
```

Comment:
i don't think this ID is used anywhere

Reply:
It was kept for consistency with other containers naming.